HelpSource/Tutorials/Getting-Started/01-Introductory-Remarks.schelp

```diff
@@ -13,6 +13,16 @@ A full list of those who have contributed to SuperCollider and its documentation
 
 https://supercollider.github.io
 
+section::Key Combinations
+
+Within this text, you will see text like "Ctrl-Enter" and "Cmd-Enter". These are key combinations. They mean: press the second key while holding down the first. 
+
+For example: "Ctrl-Enter" is a key combination of the "Ctrl" key and the "Enter" key. It means: press and hold down the "Ctrl" key and then while keeping the "Ctrl" key held down, press the "Enter" key. Once both are pressed down at the same time, you can release both keys. 
+
+"Ctrl" is a key found on most Windows and Linux computers. Mac computers have a "Ctrl" key, but when more than one key combination is given for the same instruction, the one that starts with "Cmd" is intended for Mac users. "Cmd" is short for "command" and is the key with "⌘" on it. 
+
+For key combinations with 3 or more keys, press the last key while holding down all the keys before it. For example, "Cmd-Shift-P" means: press and hold down the "Cmd" and "Shift" keys first, then press the "P" key while still holding "Cmd" and "Shift".
+
```

Comment:
Would it be clearer to explain the key combinations the other way round?

Currently it's written as "do the last thing first after doing the first things" and it a little confusing